contracts/TwoKinksInterestRateModel.sol

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+import { TimeManagerV8 } from "@venusprotocol/solidity-utilities/contracts/TimeManagerV8.sol";
+import { InterestRateModel } from "./InterestRateModel.sol";
+import { EXP_SCALE, MANTISSA_ONE } from "./lib/constants.sol";
+
+/**
+ * @title TwoKinksInterestRateModel
+ * @author Venus
+ * @notice An interest rate model with two different steep increase each after a certain utilization threshold called **kink** is reached.
+ */
+contract TwoKinksInterestRateModel is InterestRateModel, TimeManagerV8 {
+    ////////////////////// SLOPE 1 //////////////////////
+
+    /**
+     * @notice The multiplier of utilization rate per block or second that gives the slope 1 of the interest rate
+     */
+    int256 public immutable MULTIPLIER_PER_BLOCK_OR_SECOND;
```

Comment:
I will say to not make these variables immutable and make a setter also for them otherwise we will have to make new deployment every time we want to change a single value.

Reply:
I do not remember it but got your point.

Reply:
Also if we deploy new IR then we do not need to check in each VToken that the same IR is not in use by any other VToken to prevent other VTokens to update IR too. It will be more error free and time saving.